app/Http/Controllers/Chart/CragChartController.php

```diff
@@ -136,6 +136,6 @@ function climbChart($id){
             ]
         ];
 
-        return response()->json(json_encode($data));
+        return response()->json($data);
```

Comment:
btw that is not relatred with yaxis ticks scale. but just to save some CPU ticks. You encoded json twice to decode it once in *js. Now it encodes array once and on *js it has ready to use object (no need to JSON.parse). I noticed that way in few other places in oblyk code, but not sure if it was by purpose, or just some artifacts of fighting with javascript. Then I changed only here to show you it would work like that.